### PR TITLE
add new environments

### PR DESCRIPTION
**Pull request by vospitannikov**

### CODE REVIEW

File: projects/demo-app/src/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <!--ownid--><script src="https://cdn.dev.ownid.com/js/gigya-sdk.es5.js"></script><!--ownid-->
-  <!--gigya--><script src="https://cdns.gigya.com/js/gigya.js?apikey=3__r2j7EysJASGU8p1UB0Jfjf6o2e6HydoySOLx8UwerPCDpG0Vm5TA3OjBL8qW-bf"></script><!--gigya-->
+  <!--gigya--><script src="https://cdns.gigya.com/js/gigya.js?apikey=3_hOdIVleWrXNvjArcZRwHJLiGA4e6Jrcwq7RfH5nL7ZUHyI_77z43_IQrJYxLbiq_"></script><!--gigya-->
```
Comment (Tony-Pa) [inline projects/demo-app/src/index.html:11]: rollback this pls

File: projects/demo-app/src/environments/environment.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { IEnvironment, OwnIDFlow } from './i-environment';
 
 export const environment: IEnvironment = {
   production: false,
-  gigyaApiKey: '3__r2j7EysJASGU8p1UB0Jfjf6o2e6HydoySOLx8UwerPCDpG0Vm5TA3OjBL8qW-bf',
+  gigyaApiKey: '3_hOdIVleWrXNvjArcZRwHJLiGA4e6Jrcwq7RfH5nL7ZUHyI_77z43_IQrJYxLbiq_',
```
Comment (Tony-Pa) [inline projects/demo-app/src/environments/environment.ts:8]: rollback this pls

File: scripts/build_demo.sh [hunk 3]
```diff
@@ -26,4 +26,16 @@ docker build ./projects/demo-app -t ownid-demo-app:latest4
 yarn ng build demo-screens-app --configuration=$ENV
 sed -i -r -e "s/<!--gigya-->((.)*)<!--gigya-->/<script src=\"https:\/\/cdns.gigya.com\/js\/gigya.js?apikey=${GIGYA_DEMO}\"><\/script>/" projects/demo-screens-app/dist/index.html
 sed -i -r -e "s/<!--ownid-->((.)*)<!--ownid-->/<script src=\"https:\/\/cdn.${ENV}.ownid.com\/js\/gigya-sdk.es5.js\"><\/script>/" projects/demo-screens-app/dist/index.html
-docker build ./projects/demo-screens-app -t ownid-demo-screens-app:latest
\ No newline at end of file
+docker build ./projects/demo-screens-app -t ownid-demo-screens-app:latest
+
+if [ $ENV == "dev" ]; then
+  yarn ng build demo-screens-app --configuration=multi1
+  sed -i -r -e "s/<!--gigya-->((.)*)<!--gigya-->/<script src=\"https:\/\/cdns.gigya.com\/js\/gigya.js?apikey=${GIGYA_DEMO}\"><\/script>/" projects/demo-screens-app/dist/index.html
```
Comment (Tony-Pa) [inline scripts/build_demo.sh:33]: GIGYA_DEMO sets wrong api key

File: scripts/build_demo.sh [hunk 4]
```diff
@@ -26,4 +26,16 @@ docker build ./projects/demo-app -t ownid-demo-app:latest4
...
+  sed -i -r -e "s/<!--ownid-->((.)*)<!--ownid-->/<script src=\"https:\/\/cdn.${ENV}.ownid.com\/js\/gigya-sdk.es5.js\"><\/script>/" projects/demo-screens-app/dist/index.html
+  docker build ./projects/demo-screens-app -t ownid-demo-screens-app:latest2
+  
+  yarn ng build demo-screens-app --configuration=multi2
+  sed -i -r -e "s/<!--gigya-->((.)*)<!--gigya-->/<script src=\"https:\/\/cdns.gigya.com\/js\/gigya.js?apikey=${GIGYA_DEMO}\"><\/script>/" projects/demo-screens-app/dist/index.html
```
Comment (Tony-Pa) [inline scripts/build_demo.sh:38]: same

File: .github/workflows/cd.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches:
       - master
-      - develop
+      - andrii/new_configurations
```
Comment (vospitannikov) [inline .github/workflows/cd.yml:6]: !!! MUST be reverted before merge

File: .github/workflows/cd.yml [hunk 6]
```diff
@@ -48,11 +48,11 @@ jobs:
       run: yarn
     
     - name: Build Dev
-      if: github.ref == 'refs/heads/develop'
+      if: github.ref == 'refs/heads/andrii/new_configurations'
```
Comment (vospitannikov) [inline .github/workflows/cd.yml:51]: Same here

File: .github/workflows/cd.yml [hunk 7]
```diff
@@ -48,11 +48,11 @@ jobs:
...
       run: bash scripts/build_demo.sh dev 3_hOdIVleWrXNvjArcZRwHJLiGA4e6Jrcwq7RfH5nL7ZUHyI_77z43_IQrJYxLbiq_
     
     - name: Deploy Dev
-      if: github.ref == 'refs/heads/develop'
+      if: github.ref == 'refs/heads/andrii/new_configurations'
```
Comment (vospitannikov) [inline .github/workflows/cd.yml:55]: Same here

**Review by Tony-Pa — APPROVED** [review]

**Pull request merged**